pkg/route/validation/validation_test.go

```diff
@@ -1,12 +1,16 @@
 package validation
 
 import (
+	"fmt"
+	"k8s.io/apimachinery/pkg/util/validation/field"
```

Comment:
Please move the k8s.io import down to where it belongs.

Reply:
https://github.com/openshift/library-go/commit/376ae6a3fee9be3ed62f1265c81cb8f535e8a2b5#diff-51cbcb9e560e512d7dba0cd93dab386055814a27e2c7903ad93bce92c58fd18bR13